install/index.md

```diff
@@ -235,7 +236,19 @@ sections, but there is also a general troubleshooting topic.
          * Gigabyte 
 
       See [Prepare Compute Nodes](prepare_compute_nodes.md)
+
+   <a name="remove_temp_keycloak_user"></a>
```

Comment:
Please remove these anchors, instead any hyperlinks should refer to a header. If you need these anchors, I'd suggest restructuring this document to make use of sub-headers:
```markdown
### Remove Temp Keycloak User
```

This'll enable users to jump to places without making use of the anchors. The anchors were a method (originally) for adding support for Stash. Most editors don't resolve the hyper links to these anchors, since they're not technically apart of the markdown standard.